src/common/actions/auth-store.js

```diff
@@ -231,3 +233,23 @@ function checkSignedIntoStoreError(error) {
     error: true
   };
 }
+
+export function signOutOfStore(location) { // location for tests
```

Comment:
The `signOutOfStore` name may be a bit misleading, as it not only 'signs out of the store' (by clearing macaroon from localStorage), but it also redirects to logout route which clears session (signing out user from BSI completely by clearing GH token).

So I guess simply `signOut` would be more true.